docs/source/exposing.md

```diff
@@ -0,0 +1,267 @@
+# Exposing ScyllaCluster
+
+This documentation explains how the Scylla Operator exposes a ScyllaCluster on different network setups.
+The ScyllaCluster can be exposed in various network configurations, independently to clients and other nodes.
+
+:::{note}
+ScyllaCluster's can be only exposed when used ScyllaDB Enterprise version is `>=2023.1` and `>=5.2` for ScyllaDB Open Source.
+:::
+
+
+## Expose Options
+
+The ScyllaCluster created without specifying any `exposeOptions` defaults to following `exposeOptions`:
```

Comment:
```suggestion
The ScyllaCluster created without specifying any `exposeOptions` is equivalent to the following `exposeOptions`:
```
(to make sure it covers ScyllaClusters created before this change which don't technically default on the API level.)